obspy/db/scripts/indexer.py

```diff
@@ -28,17 +28,14 @@
                         unicode_literals)
 from future.builtins import *  # NOQA @UnusedWildImport
 from future.utils import native_str
-from future import standard_library
 
+import http.server
```

Comment:
Oh.. I thought I was importing directly in my clean system-Python but somehow future was doing a man-in-the-middle attack on me there.. ;-)